sc-meta-transactions.md

```diff
@@ -0,0 +1,112 @@
+* the relayer wrapps the signed transaction from the user into its own transaction by constructing and signing a meta-transaction; 
+* lastly, the Elrond protocol verifies if the inner user’s transaction is indeed signed by the user, is correctly constructed and whether it has the correct nonce. If these verifications pass, the meta-transaction is executed.
+
+On the user side, sending a meta-transaction is similar to sending a standard transaction (from, to, value ... and signature) except that instead of sending it directly to the Elrond blockchain, it sends the meta-transaction to a third party (the relayer) who will take care of the gas.
+
+One of the first dApps which needs this kind of transactions is Maiar: the official Elrond mobile wallet app. When a user registers an account on Maiar, a new or already existing Elrond address: erd1… is mapped on the Elrond blockchain to the user’s mobile phone number. The mapping process involves several transactions on Elrond blockchain, meta-transactions that from the user perspective are done by Maiar and incur for the user zero fees. More info about the mobile phone attestation and mapping process can be found here.
+
+On Maiar, a user can also choose to create a friendly username: @herotag, that is associated with his eGold (erd1…) address. The @herotag is registered on Elrond blockchain and the registration can be done through meta-transactions. This service is known as DNS service on the Elrond blockchain.
+
+Furthermore, the Elrond Standard Digital Token (ESDT) is in dire need of this functionality as well. Some users might hold only stablecoins, without any eGold to pay for the gas. Meta-transactions are thus of utmost importance for these users. Some of these users may want to pay the gas but not in eGold but in the token they hold (it depends on the relayer if it accepts payment in that token) or some relayers may want to make all the transactions free for their users (the relayer would pay the gas fee).  For example, a relayer may offer a yield farming service on Elrond network, and may be incentivised to allow its users to quickly offer liquidity to its pools even if they do not own any eGold to pay for the actual deposit transactions.
+
+The above mentioned use cases are just a few examples where meta-transactions are needed by removing the friction for users. It is up to developers to integrate this type of transactions and experiment the best onboarding experiences and different other approaches.
+
+## Flow example 
+* Alice wants to send some stablecoins directly to Bob: $10 as USDt. They both have accounts on Elrond blockchain but Alice does not have eGold. She has only USDt in the form of stablecoins.
+
+* Alice opens the wallet and signs a message that she wants to send $10 to Bob, but without any gas, because the gasPrice will be supported by a relayer.
```

Comment:
I think we should state "fee" instead of gas price. Gas price without gas limit does not have effect.